src/scss/abstracts/_variables.scss

```diff
@@ -6,6 +6,6 @@ $session-list-max-height: 31em !default; // show 6 items
 $chooser-width: 220px !default;
 $chooser-collapsed-width: 50px !default;
 
-$workbench-width: calc(100vw - #{$chooser-collapsed-width}) !default;
-$workbench-focus-width: calc(90vw - #{$chooser-collapsed-width}) !default;
-$workbench-hide-width: 10vw !default;
+$workbench-width: calc(100vw) !default;
+$workbench-focus-width: calc(85vw) !default;
+$workbench-context-width: 15vw !default;
```

Comment:
Add a comment that these widths must be in sync with the one in the TS file. Please also add a comment to the TS file.